packages/fxa-components/jest.config.js

```diff
@@ -1,9 +1,12 @@
+const { resolve } = require("path");
+
 module.exports = {
   roots: ["<rootDir>"],
   transform: {
     "^.+\\.tsx?$": "ts-jest"
   },
   moduleNameMapper: {
     "\\.(css|scss)$": "identity-obj-proxy",
+    "^@fxa-shared/(.*)$": resolve(__dirname, "../fxa-shared", "$1")
```

Comment:
Jest can now resolve the @fxa-shared module when being tested in fxa-components.